doc/Getting_Started_Writing_Clients.adoc

```diff
@@ -73,6 +73,10 @@ The servers that Yubico provides are:
 
 These servers are hosted in different places and by different organizations.
 
+Clients should send authentication requests to all of them in parallel, and
+utilize the first response recieved for added reliability should one or more
```

Comment:
Yes, I left that bit out on purpose.

Might be more confusion come to think of it, how about adding a link to one of our clients as an example instead ?

```
Clients should send authentication requests to all of them in parallel, and
utilize the first response received for added reliability should one or more
of the api servers be offline, see (https://github.com/Yubico/yubico-c-client)[Yubico C client] for technical details.
```

Yes, this is a punt until there is a better validation spec.